### ISSUE TITLE
Add track by to ng-repeats

### ISSUE THREAD

**Issue by d47853** [gchq/gaffer-tools#305]
This will mean angular no longer has to watch a whole object to guarantee uniqueness. This will result in a performance boost especially for the table, which has nested ng-repeats.

**Comment by n3101** [comment]
The UI fails to meet our Accessibility policy. It is unmaintainable and will be removed shortly.
